src/hooks/useOnMount.jsx

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useRef } from 'react';
+
+const useOnMount = (callback) => {
```

Comment:
nit: maybe add a JSDoc describing how this is intended to be used for others / future us (e.g., that the callback function passed in is what's called on mount, etc.).